dotnet/src/webdriver/StaleElementReferenceException.cs

```diff
@@ -68,7 +69,7 @@ public StaleElementReferenceException(string message, Exception innerException)
         /// </summary>
         /// <param name="message">The original message for exception</param>
         /// <returns>The final message for exception</returns>
-        protected static string GetMessage(string message)
+        protected static string GetMessage(string? message)
         {
             return message + "; " + supportMsg + supportUrl;
```

Comment:
if `message` is null then exception in runtime? something like FormatException? I mean please double check that string concatenation works with `null` (AFAIR it doesn't)

Reply:
I checked, it works. Hm, trying to recall when I saw it, and why I always use `$"{a}{b}"` now.

Reply:
That's a valid thought, I converted all the `GetMessage` method to interpolated strings.

> why I always use `$"{a}{b}"` now

On older runtimes, string interpolation always converts to `string.Format`. Sometimes it can simplify to `string.Concat`. But starting in .NET 6, there was a huge overhaul and the compiler takes care of a lot of magic for us. I think it's always better.